src/components/Gradebook/SearchControls.test.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { SearchControls } from './SearchControls';
+
+describe('SearchControls', () => {
+  let props;
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+    props = {
+      courseId: 'course-v1:edX+DEV101+T1',
+      filterValue: 'alice',
+      setGradebookState: jest.fn().mockName('setGradebookState'),
+      showSpinner: false,
+      toggleFilterDrawer: jest.fn().mockName('toggleFilterDrawer'),
+      // From Redux
```

Comment:
fwiw, for tests: I find it more helpful to group the props by type (fn vs static) vs from origin, as that makes it easier to see which things need resets, names, etc.
Also: only functions that appear directly in the JSX need names :-)